k8s/prod/common/sscs/sscs-cor-backend.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: sscs-cor-backend
+  namespace: sscs
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: sscs-cor-backend
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: sscs-cor-backend
+    version: 0.0.5
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "d6c3dd58-af16-425d-8c4a-f491e895e35d"
+      image: hmctspublic.azurecr.io/sscs-cor/backend:prod-3bc6785f
+      environment:
+        IDAM_URL: https://idam-api.platform.hmcts.net
+        PDF_SERVICE_BASE_URL: https://docmosis.platform.hmcts.net/rs/render
+        PDF_SERVICE_HEALTH_URL: https://docmosis.platform.hmcts.net/rs/status
+        PDF_SERVICE_CONVERT_URL: https://docmosis.platform.hmcts.net/rs/convert
+        IDAM_OAUTH2_REDIRECT_URL: https://sscs-case-loader-prod.service.core-compute-prod.internal
+        IDAM_SSCS_URL: https://sscs-case-loader-prod.service.core-compute-prod.internal
```

Comment:
is this right ?

Reply:
Yeh

Reply:
case loader being a job shouldn't have a end point accessible ? Is this some kind of workaround ?

Reply:
it's whitelisted in IDAM as a 'redirect_uri' it's only used to get an authentication token